app/models/enterprise.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: false
 
-require "mini_magick"
-
```

Comment:
Sorry I feel like I'm missing something. If the image has not yet been processed, would MiniMagic automatically process it? So why not keep this `require` and ensure the image processes successfully, instead of skipping over it?

Or is MiniMagick not providing any benefit here?

Reply:
We need the require statement if we reference MiniMagick in the code. But when we only reference StandardError then we don't need to know about MiniMagick. If the image needs processing then Minimagick gets loaded and if that fails we get an error and catch it correctly. That's the happy path. But if something else fails, for example AWS raises an error then the rescue statement goes through the different error classes and doesn't know what a MiniMagick error is unless we load that code first.

So the bad path is not loading Minimagick and some other error being raised.

Reply:
Ah, thank you I see now 👍 
We only needed to `require` MiniMagic so that we could make reference to a `MiniMagic` error. 

If a `MiniMagic` error does get raised, then obviously MiniMagic has been loaded because the code raised the error.